### DISCUSSION TITLE
玻璃线缆重启后失效和核电堆温bug

### DISCUSSION THREAD

**Discussion by loochickenool** [IndustrialCraftMC/IndustrialCraft2#105] [【漏洞反馈】Bugs Report]
在服务器中9月6日版本1.19.2forge ic2中 玻璃线缆在重新登陆进服务器后，在不挖掉线缆重新放置的情况下无法传输eu且储电箱无法输入或输出电力。挖掉玻璃线缆会直接消失无法掉落。
反应堆在升温至80度往上（具体不记得了）时，会导致单人客户端/服务器世界崩溃无法再次进入，在修改config文件将反应堆爆炸系数从45改为0后世可再次进入。其他小问题：ic2铜和原版铜重复，反应堆金色那个散热器合成表有问题，部分散热器（元件散热器等）和导热元件合成有问题或缺少合成表。

**Comment by AbbottMc** [comment]
ic2exp没自己的铜了已经

**Reply by loochickenool** [reply]
确实，等合成表删了就问题不大了，但是导线在重启后无法输入输出eu问题着实是很影响，而且每次都得把所有导线敲掉再重新放，挖掘且无法掉落
